package.json

```diff
@@ -17,53 +17,51 @@
     "lint:all": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
     "lint": "pnpm lint:all",
     "lint:prod": "set NODE_ENV=production&& eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
-    "precommit": "pnpm lint:prod && pnpm test && pnpm pretty:staged ",
+    "precommit": "pnpm lint:prod && pnpm test && pnpm prettier:check ",
     "prepare": "husky install",
     "prettier:check": "prettier --check --ignore-path .prettierignore .",
     "prettier:fix": "prettier --check --write --ignore-path .prettierignore .",
-    "pretty": "pretty-quick",
-    "pretty:staged": "pretty-quick --staged",
```

Comment:
If you're removing the usage of `pretty-quick` we should also remove the package from our dependencies. Though, I am a big fan of this, since it let's you easily check your staged files. When we grow this repo more, the `prettier:check` is gonna take longer and longer

Reply:
The issue is that `pretty-quick` is now almost 2 years since last update, and is now not working with the recent release of `prettier` 3

Reply:
Well, that certainly is an issue. I'm glad you're more in touch with these things than I am 😅